### PR TITLE
Confirm user emails

### PR DESCRIPTION
**Pull request by fpariani**
From here users need to confirm their email after registering to log in.

### CODE REVIEW

File: config/initializers/email_setup.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+if Rails.env.development?
+  ActionMailer::Base.delivery_method = :smtp
+  ActionMailer::Base.smtp_settings = {
+    address: 'smtp.sendgrid.net',
+    port: 25,
+    domain: 'target.com',
+    authentication: :plain,
+    user_name: 'fpariani',
```
Comment (jiv) [inline config/initializers/email_setup.rb]: You should never have credentials stored on the code. These values should be moved to ENV VARs

File: spec/requests/api/v1/users/login_spec.rb [hunk 2]
```diff
@@ -46,4 +46,19 @@
...
+  context 'with an unconfirmed email' do
+    let(:user) { create(:user) }
+    let(:params) do
+      {
+        email: user.email,
+        password: user.password
+      }
+    end
+
+    it 'returns an error after try to login' do
```
Comment (jiv) [inline spec/requests/api/v1/users/login_spec.rb]: add `ing` to `try`.

`returns an error after trying to login`

File: spec/requests/api/v1/users/logout_spec.rb [hunk 3]
```diff
@@ -25,7 +25,7 @@
       expect(response).to have_http_status(200)
     end
 
-    it "returns an error after try to logout with an expired token" do
+    it 'returns an error after try to logout with an expired token' do
```
Comment (jiv) [inline spec/requests/api/v1/users/logout_spec.rb]: same here

File: config/environments/development.rb [hunk 4]
```diff
@@ -62,4 +62,6 @@
     Bullet.rails_logger = true
     Bullet.add_footer = true
   end
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (Alxzu) [inline config/environments/development.rb]: It would be nice to have those `localhost` & `3000` inside env variables

**Pull request merged**